apps/src/dom.js

```diff
@@ -37,14 +37,14 @@ exports.getTouchEventName = function(eventName) {
 };
 
 var addEvent = function(element, eventName, handler) {
-  element.addEventListener(eventName, handler, false);
-
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
```

Comment:
Would replacing the whole handler with a technique like this maybe work? http://stackoverflow.com/a/11507558/136134